frontend/src/styles/BeforeArrival.css

```diff
@@ -0,0 +1,154 @@
+.ace-img {
+    max-width: 90%;
+    height: auto;
+    display: block;
+    margin: auto;
+    box-shadow: 0 0.5vw 1vw 0 rgba(0, 0, 0, 0.2), 0 0.7vw 2.5vw 0 rgba(0, 0, 0, 0.19);
+}
+
+img {
```

Comment:
You should change all the elements in js to have a class/className so that in css, you don't use the elements' names directly. By using `img`, `h1` etc. in css will interfere with other pages that use these elements after merging the code.